src/pages/ProfilePage.js

```diff
@@ -0,0 +1,141 @@
+import {withRouter} from 'react-router-dom';
+import styles from '../styles/styles';
+import Text from '../components/Text';
+import ONYXKEYS from '../ONYXKEYS';
+import Avatar from '../components/Avatar';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import {redirectToLastReport} from '../libs/actions/App';
+import HeaderGap from '../components/HeaderGap';
+import compose from '../libs/compose';
+import {version} from '../../package.json';
+
+const personalDetailsType = PropTypes.shape({
+    // Display name of the current user from their personal details
+    displayName: PropTypes.string,
+
+    // Avatar URL of the current user from their personal details
+    avatarURL: PropTypes.string,
+
+    // login of the current user from their personal details
+    login: PropTypes.string,
+
+    // pronouns of the current user from their personal details
+    pronouns: PropTypes.string,
+
+    // timezone of the current user from their personal details
+    timezone: PropTypes.string,
+});
+
+const matchType = PropTypes.shape({
+    params: PropTypes.shape({
+        // login passed via route /profile/:login
+        login: PropTypes.string,
+    }),
+});
+
+const propTypes = {
+    /* Onyx Props */
+    // The personal details of the person who is logged in
+    personalDetails: personalDetailsType.isRequired,
+
+    // Router details
+    match: matchType.isRequired,
+};
+
+const ProfilePage = ({personalDetails, match}) => {
+    const profileDetails = personalDetails[match.params.login];
+
+    return (
+        <>
+            <HeaderGap />
+            <HeaderWithCloseButton
+                title="Details"
+                onCloseButtonPress={redirectToLastReport}
+            />
+            <View
+                pointerEvents="box-none"
+                style={[
+                    styles.profilePageContainer,
+                ]}
+            >
+                {profileDetails ? (
+                    <View>
+                        <View style={styles.pageWrapper}>
+                            <View
+                                style={[styles.avatarLarge, styles.mb3]}
+                            >
+                                <Avatar
+                                    style={[styles.avatarLarge]}
+                                    source={profileDetails.avatarURL}
+                                />
+                            </View>
+                            <Text style={[styles.displayName, styles.mt1, styles.mb6]} numberOfLines={1}>
+                                {profileDetails.displayName
+                                    ? profileDetails.displayName
+                                    : null}
+                            </Text>
+                            {profileDetails.login && (
+                            <View style={[styles.mb6, styles.profilePageSectionContainer]}>
+                                <Text style={[styles.profilePageLabel, styles.mb2]} numberOfLines={1}>
```

Comment:
We should use the style `formLabel` for this, this way we standardize on the same field heading everywhere. cc @NikkiWines - this will make the text look bold.